src/functions-and-arrays.js

```diff
@@ -1,18 +1,82 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
```

Comment:
Here one could also use the Math.max(num1, num2) in order to find the maximum of two numbers! Find more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max) :smile: